src/components/CommentInfo/CommentInfo.tsx

```diff
@@ -1,5 +1,27 @@
 import React from 'react';
+import { Comment } from '../../types/Comment';
+import './CommentInfo.scss';
 
-export const CommentInfo: React.FC = () => (
-  <>Put the comment here</>
-);
+type CommentProps = {
```

Comment:
I think you can just name this type Props

Reply:
Agree, fix it in all cases